[F09-2] Restaurant Management System

kangmingtay opened the pull request.

stephlewyh commented:
Dear team F09-2,

Your team has shown good demo of your product with code updates. 
However you have not updated your developer guide and user guide accordingly. As mentioned on the project website, you are recommended to do this incrementally each week. 

Some housekeeping points you should note moving ahead:

Developer Guide: 
Each team member should describe at least 1 feature with a UML diagram eg. Activity, Sequence, Component diagram etc. As for UML diagrams, they should not include too low-level details or be too big, as that would go against the top-down approach for documentation. You may refer to comments I have made for the other teams regarding UML diagrams in the developer guide. 

Project Management: 
To view your own progress, you could also go to your team repo page and click 'project insights' tab for team statistics, aside from using the issue tracker and milestones to help pace your progress.

kangmingtay committed:
update member tests

kangmingtay committed:
fix checkstyle

kangmingtay committed:
fix more checkstyle

kangmingtay committed:
fix comments in PR

px1099 committed:
Add equals() method to Attendance classes

px1099 committed:
Change the StorageFileTest

kangmingtay committed:
add error messages for draftpoints command

kangmingtay committed:
add override to hasPoints in Order class

px1099 committed:
Update StorageFileTest.java

kangmingtay committed:
add case to handle negative points to redeem

px1099 committed:
Merge pull request #192 from SalsabilTasnia/version3

Added MenuRecommendationCommand

px1099 committed:
Merge pull request #185 from kangmingtay/add-phone-and-total-points

Add two new fields for the member feature

Commit:
Fix past 12 monthly revenue showing in wrong month

SalsabilTasnia committed:
Added recommendations command in developer and user guide

px1099 committed:
Remove ab from TestDataHelper (part 1)

px1099 committed:
Remove AB from TestDataHelper (part 2)

px1099 committed:
Remove AB from LogicTest.java

px1099 committed:
Merge branch 'master' into remove-AB

px1099 committed:
Fix employee and attendance bugs

px1099 committed:
Change Member hashing

px1099 committed:
Remove AB from AdaptedRms.java

px1099 committed:
Remove AB from Rms.java

px1099 committed:
Remove AdaptedPerson.java

px1099 committed:
Remove Person.java and UniquePersonList.java

px1099 committed:
Remove AB from Command.java

px1099 committed:
Remove AB from CommandResult.java

px1099 committed:
Remove AB from Logic.java

px1099 committed:
Remove AB from Formatter.java

px1099 committed:
Remove AB from MainWindow.java

px1099 committed:
Remove Person data

px1099 committed:
Remove AB from codes, comments, strings

px1099 committed:
Remove 2 testcases: clockin_success and clockout_success

px1099 committed:
Fix checkStyle

SalsabilTasnia committed:
Improved the Portfolio

kangmingtay committed:
change delete menu command format

kangmingtay committed:
change display of listmenu and invalid addmenu command help message

kangmingtay committed:
change command format of listmember

kangmingtay committed:
change display of employee commands

kangmingtay committed:
change invalid addcommand message format

kangmingtay committed:
update tests

kangmingtay committed:
Merge pull request #196 from px1099/remove-AB

Remove addressbook from project & remove clockin, clockout test cases

kangmingtay committed:
Merge branch 'master' into standardise-display-command-formats

kianhong95 committed:
Merge pull request #197 from kangmingtay/standardise-display-command-formats

Standardise display command formats

SalsabilTasnia committed:
removed viewallmenu command
added test cases for showmainmenu and menurecommendations commands
Modified the User Guide and Developer Guide accordingly

kianhong95 committed:
Merge branch 'master' into EditCommand

kianhong95 committed:
Changed EmployeeEditCommand to work with actual empty String instead of an arbitrary empty String

kianhong95 committed:
Merge branch 'master' into JUnitTests

SalsabilTasnia committed:
changed the price regex to restrict price to be less than $10,000 and changed the user guide accordingly

SalsabilTasnia committed:
resolved merge conflict

SalsabilTasnia committed:
updated user guide for the price regex

SalsabilTasnia committed:
resolved check style error

SalsabilTasnia committed:
removed all unused methods and commented methods

SalsabilTasnia committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F09-2/main into version3

kangmingtay committed:
change price regex to limit price input

px1099 committed:
Merge pull request #198 from kianhong95/EditCommand

Changed how EmployeeEditCommand works

kangmingtay committed:
add limit to currentPoints and totalPoints

kangmingtay committed:
change message price constraints message

kangmingtay committed:
change message phrasing

kangmingtay committed:
increase cap limit for current and total points

px1099 committed:
Merge pull request #200 from kangmingtay/add-parameters-for-price-input

add exceptions to handle overflow bugs in price

kangmingtay committed:
Merge branch 'master' into add-total-points-limit

kangmingtay committed:
Merge branch 'add-total-points-limit' of https://github.com/kangmingtay/main into add-total-points-limit

Commit:
Merge branch 'master' into topic-statsFunction

SalsabilTasnia committed:
commit

SalsabilTasnia committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F09-2/main into version3

Commit:
Add test for utils

kianhong95 committed:
Merge branch 'master' into JUnitTests

kangmingtay committed:
fix typo errors

kangmingtay committed:
reformat help command

kangmingtay committed:
add link to reposense for PPP

px1099 committed:
Merge pull request #202 from SalsabilTasnia/version3

Updated PortFolio, User Guide and Developer Guide, removed Viewall Command and added test cases for menu

Commit:
Separate few methods

kangmingtay committed:
fix checkstyle

Commit:
Move message to static final

px1099 committed:
Change validation regex of draftdish command to limit the quantity input

kangmingtay committed:
update Ui picture with latest Ui display

kangmingtay committed:
update PPP

px1099 committed:
Change test cases based on the new implementation

px1099 committed:
Minor message change

px1099 committed:
Update UG

Commit:
Tidy error message

px1099 committed:
Clean getAsText() in Member.java

Commit:
Add logic test for statistics command

Commit:
Add method to generate statistics command

Commit:
Add parser test for statistics help command

px1099 committed:
Clean getAsText() in Menu.java

AngWM committed:
Merge pull request #205 from px1099/limit-quantity

Limit quantity of draftdish command

Commit:
Fix checkstyle errors

Commit:
Fix wording in message usage

kangmingtay committed:
update PPP

px1099 committed:
Reformat getAsText() in Order.java

px1099 committed:
Change confirmdraft success message

px1099 committed:
Change the toString() method of Menu.java

kangmingtay committed:
fix formating error

px1099 committed:
Merge branch 'master' into clean-show-as-text

kangmingtay committed:
update formatting

kangmingtay committed:
update formatting

kangmingtay committed:
update formatting of intro

kangmingtay committed:
Merge branch 'master' into UGDG

kianhong95 committed:
Fixed tests for clockin and clockout command and changed timing displayed for clockin and clockout

kianhong95 committed:
Merge branch 'master' into JUnitTests

px1099 committed:
Merge pull request #204 from kangmingtay/UGDG

Minor modifications to UI, UGDG and PPP

kianhong95 committed:
Fixed CheckStyle erro

px1099 committed:
Update StatsMenuCommand.java

Update based on the review I made

px1099 committed:
Merge pull request #203 from AngWM/topic-statsFunction

Fix bugs and add test case

px1099 committed:
Show the entered command in the result

px1099 committed:
Merge pull request #207 from kianhong95/JUnitTests

Fixed tests for clocking and clockout command and changed timing displays for both commands

px1099 committed:
Make the Parser accept insensitive command word

px1099 committed:
Fix checkStyle

SalsabilTasnia committed:
Merge pull request #209 from px1099/command-case-insensitive

Make the Parser accept insensitive command word

SalsabilTasnia committed:
Merge pull request #208 from px1099/improvement

Show the entered command in the result

kangmingtay committed:
Merge branch 'master' into add-total-points-limit

px1099 committed:
Refactor the format of the confirmdraft message

SalsabilTasnia committed:
Merge pull request #206 from px1099/clean-show-as-text

Cleanup the showAsText() methods of data classes

kangmingtay committed:
add tests for draft points

kangmingtay committed:
update member tests

kangmingtay committed:
update tests and remove redundant sections

px1099 committed:
Improve draftdish to change multiple dish quantity at the same time

px1099 committed:
New exception for duplicate index in draftdish command

px1099 committed:
Remove old regex for draftdish

kangmingtay committed:
update test for update points

px1099 committed:
Update UG

px1099 committed:
Add/edit test cases for draftdish

SalsabilTasnia committed:
commit

SalsabilTasnia committed:
resolved merge conflicts and updated the user guide and developer guide to make instructions clearer

px1099 committed:
Manual code cleaning

px1099 committed:
Automatic code cleaning

SalsabilTasnia committed:
updated the ppp and added author tags to the relevant classes and methods

SalsabilTasnia committed:
resolved checkstyle errors

SalsabilTasnia committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F09-2/main into version3

SalsabilTasnia committed:
updated ppp

SalsabilTasnia committed:
resolved merge conflicts

px1099 committed:
Code clean up based on warnings of IntelliJ

px1099 committed:
Change spliting regex

px1099 committed:
Fixs failed tests

px1099 committed:
Fixs checkStyle

px1099 committed:
Improve code quality based on Codacy advices (part 1)

px1099 committed:
Resolve more error prone issues

SalsabilTasnia committed:
changed ug and ppp

SalsabilTasnia committed:
made changes to the UG

SalsabilTasnia committed:
made changes to ppp and updated diagram for Data Component

kangmingtay committed:
Merge pull request #210 from px1099/improve-draft-dish-command

Improve draft dish command

px1099 committed:
Merge branch 'master' into improve-code-quality

px1099 committed:
Merge pull request #201 from kangmingtay/add-total-points-limit

add limit to currentPoints and totalPoints to prevent overflow

px1099 committed:
Merge branch 'master' into improve-code-quality (again)

SalsabilTasnia committed:
commit

SalsabilTasnia committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F09-2/main into version3

kangmingtay committed:
add introductory statement to PPP

SalsabilTasnia committed:
update ppp

kangmingtay committed:
change header from intro to about myself

SalsabilTasnia committed:
commit

SalsabilTasnia committed:
commit

SalsabilTasnia committed:
merge resolve

SalsabilTasnia committed:
merge resolve

SalsabilTasnia committed:
added naming constraints

SalsabilTasnia committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F09-2/main into version3

SalsabilTasnia committed:
commit

SalsabilTasnia committed:
resolved conflicts

kangmingtay committed:
update contributions

kangmingtay committed:
fix some typo errors in UG

kangmingtay committed:
Merge pull request #211 from px1099/improve-code-quality

Improve code quality based on IntelliJ and Codacy warnings

kangmingtay committed:
remove exclaimation marks in intro

Commit:
Update portfolio

Commit:
Update DG

Commit:
Add images

px1099 committed:
tag authors in Command classes

px1099 committed:
Add author tags to Common classes

kianhong95 committed:
Added author tags

kianhong95 committed:
Edited author tags to the correct formatting

kianhong95 committed:
Merge branch 'master' into ProjectPortfolio

kianhong95 committed:
Merge pull request #214 from SalsabilTasnia/version3

Added author tags, removed unused codes and updated PPP and UG and DG

px1099 committed:
Add tagging for employee data classes

px1099 committed:
Remove merge conflict

Commit:
Add intro

px1099 committed:
Merge pull request #212 from kangmingtay/UGDG

Edit Kang Ming's PPP

Commit:
Fix typo

px1099 committed:
Merge pull request #216 from px1099/fix-conflict

Remove merge conflict

px1099 committed:
Merge pull request #215 from kianhong95/AuthorTags

Added author tags

px1099 committed:
Merge branch 'master' into tag-author

Commit:
Update code contributed link

px1099 committed:
Edit command author tags

px1099 committed:
Employee authors tags

Commit:
Merge branch 'master' into topic-addPortfolio

px1099 committed:
Tag authors Data classes

px1099 committed:
Add author tags in Logic & Parser

kianhong95 committed:
Merge branch 'master' into MiscEdits

px1099 committed:
Add authors tags to adapted classes

px1099 committed:
Add author tags to UI classes

kianhong95 committed:
Change EmployeeEmail to extend from Email

kianhong95 committed:
Changed EmployeePhone to only accept 8-digit numbers

kianhong95 committed:
Merge branch 'master' into UserGuide

px1099 committed:
Update author tags to testcases

kangmingtay committed:
update DG

kangmingtay committed:
Merge pull request #219 from SalsabilTasnia/version3

Edited PPP, UG and Data Component Diagram

Commit:
Add implementation for statistics

Commit:
Merge branch 'master' into topic-addPortfolio

Commit:
Add space

Commit:
Update portfolio

SalsabilTasnia committed:
Merge pull request #213 from AngWM/topic-addPortfolio

Update docs such as PPP, UG, DG

px1099 committed:
Fix test cases

px1099 committed:
Merge branch 'master' into tag-author

kangmingtay committed:
change the data component diagram

kangmingtay committed:
add implementation of draft points command

Commit:
Add screenshots

Commit:
Add more info

kianhong95 committed:
Updated PPP

kangmingtay committed:
resolve merge conflicts

kianhong95 committed:
Merge pull request #222 from AngWM/topic-addPortfolio

Update docs

AngWM committed:
Merge pull request #223 from kianhong95/ProjectPortfolio

Updated Project Portfolio

SalsabilTasnia committed:
Merge pull request #220 from kangmingtay/UGDG

Modified Developer's Guide

kianhong95 committed:
Merge pull request #218 from px1099/tag-author

Tag authors in project

kianhong95 committed:
Merge branch 'master' into MiscEdits

kianhong95 committed:
Merge branch 'master' into UserGuide

AngWM committed:
Merge pull request #221 from SalsabilTasnia/version3

Update PPP

kangmingtay committed:
uncommented some parts of the DG

kangmingtay committed:
remove headless mode from tests

px1099 committed:
Update ppp

kangmingtay committed:
change regex for name

px1099 committed:
Update ppp

kangmingtay committed:
change regex for Email

kangmingtay committed:
fix display

kangmingtay committed:
Merge pull request #217 from kianhong95/MiscEdits

EmployeeEmail now extends Email and EmployeePhone regex has been changed

kianhong95 committed:
Updated User Guide

AngWM committed:
Merge pull request #224 from kangmingtay/devguide-changes

more changes to DG

kangmingtay committed:
Merge pull request #225 from kianhong95/UserGuide

Updated User Guide

kangmingtay committed:
made some changes to member section

px1099 committed:
Update build.gradle and ascii doc template

px1099 committed:
Merge pull request #228 from px1099/update-ppp

Update px1099's portfolio

kianhong95 committed:
Merge branch 'master' into MiscEdits

AngWM committed:
Merge pull request #227 from SalsabilTasnia/version3

update menu name limitations and UG

kangmingtay committed:
change typo on UG

kianhong95 committed:
Updated constraints for add employee command

AngWM committed:
Merge pull request #226 from kangmingtay/UGDG

made some changes to member section

px1099 committed:
Update README and ContactUs

px1099 committed:
Merge pull request #230 from px1099/contact-us

Update README and ContactUs

Commit:
Update format

kangmingtay committed:
fix inconsistencies in PR

kangmingtay committed:
Merge pull request #231 from kangmingtay/UGDG

fix inconsistencies in PR

Commit:
Merge branch 'master' into topic-addPortfolio

Commit:
Update link

AngWM committed:
Merge pull request #229 from kianhong95/MiscEdits

Updated constraints for Add Employee Command

px1099 committed:
Update ui.png and version number

kianhong95 committed:
Merge pull request #232 from AngWM/topic-addPortfolio

Topic add portfolio

px1099 committed:
Merge pull request #233 from px1099/update-version

Update ui.png and version number

kianhong95 committed:
Minor edits to the PPP

px1099 committed:
Fix ContactUs

kianhong95 committed:
Merge pull request #234 from kianhong95/MiscEdits

Minor edits to the PPP

px1099 committed:
Update DeveloperGuide

px1099 committed:
Change Title color

px1099 committed:
Merge pull request #235 from px1099/update-ascii-doc

Update ascii doc
